src/resources/EnvironmentStatusesWithStages.yaml

```
@@ -0,0 +1,20 @@
+get:
+  summary: 'Get environment statuses with stages'
+  operationId: getEnvironmentStatusesWithStages
+  parameters:
+    - $ref: '../parameters/path/environmentId.yaml'
+  tags:
+    - Environment Main Calls
+  responses:
+    '200':
+      description: 'Get statuses'
```

Description should be updated as well:
```suggestion
      description: 'Get statuses with stages'
```